src/test/regress/input/multi_copy.source

```diff
@@ -497,7 +497,19 @@ SELECT shardid, nodename, nodeport
 	WHERE logicalrelid = 'numbers_append'::regclass order by placementid;
 
 -- add the node back
-SET client_min_messages TO ERROR;
+-- before adding the node, add pg_dist_object entry for tables created with
+-- master_create_distributed_table as we don't have the entry for them.
+
+INSERT INTO citus.pg_dist_object(classid, objid, objsubid) values('pg_class'::regclass::oid, 'objects'::regclass::oid, 0);
```

Comment:
This looks very unusual :/  Can't we simply convert these tables to `create_distributed_table`? I think almost half of the tables are already here, the others should be easy to find?

(We'll need to handle this case for upgrade to Citus 11, but that's a separate discussion.)

Reply:
Move it to master_create_distributed_table.

Reply:
Moving it to the master_create_distributed_table affects lots of test where we manually check existency/non-existency of shell tables/dist object on nodes. So I'm keeping it as it is. Let's handle this one with a subsequent PR as well.